python/paddle/fluid/contrib/slim/quantization/post_training_quantization.py

```diff
@@ -118,7 +118,7 @@ def _apply_pass(scope,
 class PostTrainingQuantization(object):
```

Comment:
Does this file have any change ? It looks only adding some spaces ?

Reply:
No. It is here by mistake.